packages/frontend/src/state/reducers/drafts/actions.ts

```diff
@@ -0,0 +1,12 @@
+import type { SerializableObject } from "@carrot-kpi/react";
+import { createAction } from "@reduxjs/toolkit";
+
+export const addDraft = createAction<{
+    draftId: number;
```

Comment:
Since this is already the draft entity does it make sense for this to just be called `id`?

Reply:
Fixed in ec5ed1be03f556cb81c2982b3f68bc703052ae94.